modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -196,6 +205,28 @@ export default {
       } else {
         return [];
       }
+    },
+    needPublishButton() {
```

Comment:
A thing that autopublishes isn't *un-*publishable. I think "needPublishButton" is the right message. The case on L341 is about the button, because it is about determining whether we need a basis for comparison to decide when the button should be activated.

Reply:
Then `manuallyPublished`, maybe? `needPublishButton` communicates to me that this controls whether the publish button exists or is active. This property is controlling way more business logic, including whether to run the `publish` method.